frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -556,10 +557,23 @@ export class TransformResourceData {
     );
   };
 
+  getActiveStatefulSets = (ss: K8sResourceKind): K8sResourceKind[] => {
+    const { statefulSets } = this.resources;
+    const ownedRS = _.filter(statefulSets?.data, (f) => f.metadata.name === ss.metadata.name);
+    return _.filter(ownedRS, (rs) => _.get(rs, 'status.replicas'));
+  };
+
   public getReplicaSetsForResource = (deployment: K8sResourceKind): PodControllerOverviewItem[] => {
     const replicaSets = this.getActiveReplicaSets(deployment);
     return sortReplicaSetsByRevision(replicaSets).map((rs) =>
-      getIdledStatus(this.toReplicaSetItem(rs), deployment),
+      getIdledStatus(this.toResourceItem(rs, ReplicaSetModel), deployment),
+    );
+  };
+
+  public getStatefulSetsForResource = (ss: K8sResourceKind): PodControllerOverviewItem[] => {
```

Comment:
I dont think this func is quite right. 

```
Deployment -> ReplicaSets -> Pods
StatefulSet -> ReplicaSet -> Pods
```
Stateful sets are not "for" another resource type.